sys-devel/llvm/llvm12-12.0.0.recipe

```diff
@@ -551,6 +551,11 @@ INSTALL()
 		liblld* \
 		libRemarks
 
+	# adjust cmake library paths because prepareInstalledDevelLibs moved them
+	sed -i 's,\${_IMPORT_PREFIX}/lib/,\${_IMPORT_PREFIX}/develop/lib/,' \
```

Comment:
I "think" this won't work on 32bit (didn't check there yet) because the libraries are installed in */develop/lib/x86 ?
If so a check for targetarchitecture would be needed?

Reply:
Probably.  I haven't looked at the package on 32 bit but I would guess that it's still broken.

Reply:
> I "think" this won't work on 32bit (didn't check there yet) because the libraries are installed in */develop/lib/x86 ? If so a check for targetarchitecture would be needed?

I finally got around to looking at this so I could extend the workaround for x86 but it isn't needed.  The paths in the files are correct and I was able to use it to compile some other software.  The only immediate change that I need to make is that I missed one of the files.  I need to add $libDir/cmake/lld/LLDTargets-release.cmake